### PR TITLE
Add blacklist

### PR DESCRIPTION
**Pull request by Raycoms**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
-
- Add block blacklist, to allow serverowners and packmakers to exclude blocks that allow duping
-

Review please

### CODE REVIEW

**Review by someaddons — APPROVED** [review]

File: src/main/java/com/ldtteam/structurize/blueprints/v1/BlueprintUtil.java [hunk 1]
```diff
@@ -74,6 +75,10 @@ public static Blueprint createBlueprint(
         for (final BlockPos mutablePos : BlockPos.betweenClosed(pos, pos.offset(sizeX - 1, sizeY - 1, sizeZ - 1)))
         {
             BlockState state = world.getBlockState(mutablePos);
+            if (state.is(ModTags.BLUEPRINT_BLACKLIST))
```
Comment (Nightenom) [inline src/main/java/com/ldtteam/structurize/blueprints/v1/BlueprintUtil.java:78]: uhh what? what's the code path of this method? we definitely want to save what we scan fully, such blacklisting should take place during structure placing

Comment (Raycoms) [reply src/main/java/com/ldtteam/structurize/blueprints/v1/BlueprintUtil.java:78]: I'm doing both. Preventing people to scan blacklisted blocks (also to make it harder to try to bypass these things in past versions. Gotta scan in past and place in past!)

Comment (Nightenom) [reply src/main/java/com/ldtteam/structurize/blueprints/v1/BlueprintUtil.java:78]: This is hella flawed imho, especially you dont care about what you scan/place in creative, also you usually don't care if you play singleplayer (regardless of pack maker blacklisting or not), makes really no sense for me to do this during scanning

Comment (Raycoms) [reply src/main/java/com/ldtteam/structurize/blueprints/v1/BlueprintUtil.java:78]: This is not only for exploity stuff, but also for stuff that crashes, etc (doesn't work well at all when scanned).

**Pull request merged**